lom/src/engine/engine.go

```diff
@@ -0,0 +1,207 @@
+package engine
+
+import (
+    "bytes"
+    "flag"
+    . "lom/src/lib/lomcommon"
```

Comment:
Isn't it better to just use packages data prefixed by its name instead of importing everything here ? Overtime we may have lomcommon and may be some other helper packages and using same names inside. package prefix helps in that case saving time to resolve compilation issues.

Reply:
lomcommon is intended for common APIs use across. 
Tagging everywhere just clutters and affects code reading.